pkg/util/go_util.go

```diff
@@ -60,6 +74,12 @@ func RetryDuring(attempts int, timeout time.Duration, sleep time.Duration, myfun
 			return nil
 		}
 
+		if _, skipRetry := err.(*ErrorSkipRetry); skipRetry {
+			err = fmt.Errorf("failing without retries: %v", err)
+			glog.Error(err)
```

Comment:
Nit: I think this log is redundant.

Reply:
Thanks for the review @ading1977. Removed this and other similar logs in this method.